### PR TITLE
[MM-37726] - Self-managed: Update email template to new branding - Invitation email

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This email updates the invitation email template to the new branding

Screenshot 2021-08-13 at 15 04 07

#### Ticket Link
[MM-37726](https://mattermost.atlassian.net/browse/MM-37726)

#### Release Note

```release-note
NONE

```

### CODE REVIEW

**Comment by neallred** [comment]
Should `templates/cloud_welcome_email.html` also be updated to use the changed image?

**Comment by AGMETEOR** [comment]
> Should `templates/cloud_welcome_email.html` also be updated to use the changed image?

Will update in a separate ticket

File: templates/partials/style.mjml [hunk 1]
```diff
@@ -24,13 +24,13 @@
...
     letter-spacing: -0.01em !important;
-    color: #3D3C40 !important;
+    color: #3F4350 !important;
   }
 
   .subTitle div {
     font-size: 16px !important;
     line-height: 24px !important;
-    color: rgba(61, 60, 64, 0.64) !important;
+    color: rgba(63, 67, 80, 1) !important;
```
Comment (pablovelezvidal) [inline templates/partials/style.mjml:33]: if opacity is 100%, no need to add the rgba, rgb without the one is enough.

File: templates/invite_body.html [hunk 2]
```diff
@@ -450,24 +454,39 @@
...
+                                    </td>
+                                  </tr>
+                                  {{if .MessageURL}}
+                                  <tr>
+                                    <td align="center" vertical-align="middle" class="messageButton" style="font-size:0px;padding:16px 0px 0px 0px;word-break:break-word;">
+                                      <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse:separate;line-height:100%;">
+                                        <tr>
+                                          <td align="center" bgcolor="#FFFFFF" role="presentation" style="border:none;border-radius:4px;cursor:auto;mso-padding-alt:10px 25px;background:#FFFFFF;" valign="middle">
+                                            <a href="{{.MessageURL}}" style="display:inline-block;background:#FFFFFF;color:#ffffff;font-family:Open Sans, sans-serif;font-size:13px;font-weight:normal;line-height:120%;margin:0;text-decoration:none;text-transform:none;padding:10px 25px;mso-padding-alt:0px;border-radius:4px;" target="_blank">
```
Comment (pablovelezvidal) [inline templates/invite_body.html:481]: is this template auto-generated? wondering if there is no other way to set this styles in a way that is easier to read and thus to maintain.

Comment (AGMETEOR) [reply templates/invite_body.html:481]: @pablovelezvidal  yes, this is autogenerated

**Review by pablovelezvidal — APPROVED** [review]
LGTM! :shipit:

**Review by neallred — APPROVED** [review]
Looks good to me!

**Pull request merged**